core/tests/performance_framework/perf_services.py

```diff
@@ -148,11 +203,14 @@ def get_page_timings_from_cached_session(self, page_url):
         """
```

Comment:
There are different tests for both size and for timings. Now, for every test either only size or timing is required. Hence, even if we do it together in the same method it would be worthless.